packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -433,6 +433,13 @@ export class GumbyController {
         } else {
             transformByQState.setMultipleDiffs(false)
         }
+        await telemetry.codeTransform_submitSelection.run(async () => {
+            telemetry.codeTransform_submitSelection.emit({
```

Comment:
You can put everything inside this `run` block just like we do below, then `emit` isn't needed, we can just use `telemetry.record()` as done below.